src/modules/extendedCallWindow/i18n/it_IT.js

```diff
@@ -109,7 +109,7 @@ module.exports = {
             [/^Volanti?$/]: [7, 16, 15, 21],
             [/^Furgon[ie] artificieri$/]: [22],
             [/^Elicotter[io] della polizia?$/]: [13],
-            [/^furgon[ae] antisommossa$/]: [14],
+            [/^Unit(à)? antisommossa$/]: [14],
```

Comment:
What are the possible values? Now `Unit` and `Unità` do match. Is it what you want?

Reply:
the only one should be Unità

Reply:
Then you can use `/^Unità antisommossa$/`. You don't need any special characters.

Reply:
would this apply to `/^Unit(à)? cinofila antidroga$/` as well? as it only needs to match Unità

Reply:
Yes. The `?` marks that the character (or group if encloses in parantheses `()`) is optional.
`Hello? World` matches `Hello World` and `Hell World`.
`He(llo)? World` matches `Hello World` and `He World`.

Reply:
Oh, thanks, appreciate it